src/components/accession2/view/Accession2View.tsx

```diff
@@ -312,20 +312,20 @@ export default function Accession2View(props: Accession2ViewProps): JSX.Element
               onEdit={() => setOpenNewViabilityTest(true)}
             />
           )}
-          {openNewViabilityTest && (
```

Comment:
why remove this?

Reply:
Beacause the new modal can't open if that one don't exist (after closing it)

Reply:
Yes, also tried that, but the new modal is opening behind :(